pos_payment_terminal/models/account_journal.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Aurélien DUMAINE
+# © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    payment_mode = fields.Selection(
```

Comment:
Maybe a more specific field name should be used (something like `terminal_payment_mode`) as this is a model used for other workflows.

Reply:
Or maybe migrating to payment.mode model as in https://github.com/OCA/bank-payment/tree/10.0/account_payment_mode which could be more correct